### ISSUE TITLE
Features

### ISSUE THREAD

**Issue by CCRS-WEB** [hirokiky/pilemd#18]
Hi,

I'm trying PileMD, considering use it as my full time notes taker.
Here's a quick list of features would like to see in it.

- resize folders panel width and notes panel width
- expand/collapse RACK 
- hide the preview of notes 
- sub-folder for folder
- side by side edit/preview mode
- help for markdown syntax
- highlight for language
- enable translation of the app
- moire customization, ex: change/choose fonts and color for editor and panel items

Regards,

**Comment by hirokiky** [comment]
Thanks. 

Nice:

* resize folders panel width and notes panel width
    * Great. I imagine it's like "Notes" app on OSX.
* hide the preview of notes
    * Changing way to display notes list. Nice, it will be like Preview/Only Titles 
* help for markdown syntax
    * I think so. Some one also said it (face to face)
* ~~moire~~ more customization, ex: change/choose fonts and color for editor and panel items
    * I think so. Most of people want it. (Need great PR)

Not so important:

* expand/collapse RACK
    * Basically I think people will have 20 ~ 30 folders, it can be displayed.
* enable translation of the app
    * Before translation, make the app simple.

Won't:

* side by side edit/preview mode
    * PileMd's edit mode should be like preview. I know today's PileMd is not perfect, so it's difficult to see images on edit mode. We need switch edit/preview over and over again. We need nice solution (but not side by side)
* sub-folder for folder
    * PileMd won't support tree of folders. It makes the app complex and basically human-being will make folder structures messy without any kinds of restrictions. If you want to use tree of folders, use Atom editor. or, If you have great solution (tree folder view, but people won't make them messy), that's the best.

**Comment by CCRS-WEB** [comment]
Thank you for detailled answers, see below some comments.

***hide the preview of notes**
_Changing way to display notes list. Nice, it will be like Preview/Only Titles_
Yes, this will make search more clean, often i don't need to preview content. 

**expand/collapse RACK**
_Basically I think people will have 20 ~ 30 folders, it can be displayed._

Personnaly, i have more than 30 folders cause i work on multiple projects and have few personnal notes about dev.
Basically, client or entities = rack and "projects" = folder
The numbers of folders grows quickly over 30, so expand/collapse can make PileMD  view and usage cleaner.

Regards,
